.travis.yml

```diff
@@ -35,6 +35,38 @@ matrix:
           php: 7.3
           env: TEST_COMMAND="./vendor/bin/phpunit --group=NothingInstalled" DEPENDENCIES="phpunit/phpunit:^7.5"
 
+        - name: Test Install
+          php: 7.3
+          install:
+              - |
+                # install_test is a helper to create folded reports (From Symfony)
+                install_test () {
+                    local title="$1 \"$2\" ..."
+                    local fold=$(date +%s%N)
+                    echo -e "travis_fold:start:$fold"
+                    echo -e "\\e[1;34m$title\\e[0m"
+                    echo "./tests/install.sh \"$1\" \"$2\" \"$3\""
+                    ./tests/install.sh "$1" "$2" "$3" 2>&1
+                    local ok=$?
+                    (exit $ok) &&
+                        echo -e "\\e[32mOK\\e[0m $title\\n\\ntravis_fold:end:$fold" ||
+                        echo -e "\\e[41mKO\\e[0m $title\\n"
+                    (exit $ok)
+                }
+                export -f install_test
```

Comment:
could we add this as a bash file to tests folder, or `.travis` folder? no big deal, but it seems a bit much for the travis configuration file

Reply:
I dont think so. Im not sure how you can "export" a function in bash/travis.

Reply:
ah i see, bash function, not bash script. i guess thats fine then. maybe bash `source`-ing the file would work, but i will merge as is and tag another patch release